### PR TITLE
feat: creation of the table component

### PR DESCRIPTION
**Pull request by GusBaamonde**
table-component

### CODE REVIEW

File: assets/scss/modules/_tables.scss [hunk 1]
```diff
@@ -1 +1,117 @@
...
+///           Content 1
+///         </td>
+///         <td>
+///           Content 2
+///         </td>
+///       </tr>
+///     </tbody>
+///   </table>
 
+@mixin dp-c-table {
```
Comment (cbernat) [inline assets/scss/modules/_tables.scss]: Why use a mixin here instead of a simple class? I'm asking out of curiosity,

**Comment by cbernat** [comment]
Test this please.

File: assets/scss/modules/_tables.scss [hunk 2]
```diff
@@ -1 +1,117 @@
...
+
+    th {
+      background: $dp-color-white;
+      border-top: 1px solid $dp-color-silver;
+      border-bottom: 1px dashed $dp-color-lightgrey;
+      font-size: $dp-sizing--lvl1;
+      font-weight: normal;
+      color: $dp-color-grey;
+      height: 40px;
+      line-height: 18px;
```
Comment (andresmoschini) [inline assets/scss/modules/_tables.scss]: There are no constants for these values?

File: assets/scss/modules/_tables.scss [hunk 3]
```diff
@@ -1 +1,117 @@
...
+        &:hover {
+          background: rgba(251, 178, 36, 0.1);
+        }
+      }
+
+      td {
+        color: $dp-color-grey;
+        border: 0;
+        box-shadow: $dp-spaces--lv0 -1px $dp-spaces--lv0 $dp-spaces--lv0 $dp-color-silver inset;
+        height: 40px;
```
Comment (andresmoschini) [inline assets/scss/modules/_tables.scss]: constant?

File: assets/templates/index.html [hunk 4]
```diff
@@ -383,6 +383,13 @@ <h2>Botones</h2>
             </div>
           </div><!-- button-card -->
         </main>
+      <section class="section-module">
+        <h2>Table component</h2>
+        <!-- build:include partials/_table-component.html -->
```
Comment (andresmoschini) [inline assets/templates/index.html]: ❤️

File: assets/templates/partials/_table-component.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        <td>Content 5</td>
+    </tr>
+    <tr>
+      <td>Content 1</td>
+      <td>Content 2</td>
+      <td>Content 3</td>
+      <td>Content 4</td>
+      <td>Content 5</td>
+    </tr>
+    <tr>
```
Comment (andresmoschini) [inline assets/templates/partials/_table-component.html]: Something weird happened with indentation here.

File: assets/scss/modules/_tables.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/modules/_tables.scss]: @cbernat  as we chat, the mixins can receive parameters and arguments, allows us to dynamising our css, while the loose classes do not. This allows us to reuse our code.

File: assets/scss/modules/_tables.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/modules/_tables.scss]: the line-height is used according to the size of the typography, we do not have all the line-height defined

File: assets/templates/partials/_table-component.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
+<table class="dp-c-table">
```
Comment (andresmoschini) [inline assets/templates/partials/_table-component.html]: How does it define the values for _$bg-color-zebra_, _$font-size-table_ and _$bg-color-hover_ mixing parameters?

File: assets/scss/modules/_tables.scss (hunk 2 above)
Comment (andresmoschini) [reply assets/scss/modules/_tables.scss]: In that case, it should not be a kind of calculus? 

I mean, suppose that you change the value of _$dp-sizing--lvl1_, for example from _15px_ to _60px_, in that case, a _line-height_ of _18px_ will be wrong.

I am missing something?

**Comment by andresmoschini** [comment]
> Test this please.

@GusBaamonde that message was for Jenkins bot, see this: https://makingsense.slack.com/archives/GGN03EBDM/p1562931800042500?thread_ts=1562931766.042400&cid=GGN03EBDM

File: assets/scss/helpers/_colors.scss [hunk 7]
```diff
@@ -29,3 +29,4 @@ $dp-bg-modal: #F7F7F7;
 $dp-bg-overlay: rgba(34, 34, 34, 0.95);
 $dp-color-brown-brightness: rgba(48,33,0,.2);
 $dp-color-wild-sand: #F6F6F6;
+$dp-color-hover-table: #fff7e8;
```
Comment (cbernat) [inline assets/scss/helpers/_colors.scss]: I think this should be in capitals as the other colors right?

**Comment by cbernat** [comment]
> > Test this please.
> 
> @GusBaamonde that message was for Jenkins bot, see this: https://makingsense.slack.com/archives/GGN03EBDM/p1562931800042500?thread_ts=1562931766.042400&cid=GGN03EBDM

Yes I supposed you understood. It was for triggering a new build, regarding the issue of space in the server.

**Comment by andresmoschini** [comment]
> > > *PIP* this please.
> > 
> > 
> > @GusBaamonde that message was for Jenkins bot, see this: https://makingsense.slack.com/archives/GGN03EBDM/p1562931800042500?thread_ts=1562931766.042400&cid=GGN03EBDM
> 
> Yes I supposed you understood. It was for triggering a new build, regarding the issue of space in the server.

Each time that we mention that sentence, Jenkins has to work.

**Comment by GusBaamonde** [comment]
Test this please.

**Pull request merged**